composer.json

```
@@ -4,10 +4,10 @@
   "description": "Mink extension for Google Place Autocomplete",
   "license": "MIT",
   "require": {
-    "php": ">=5.6",
+    "php": ">=7.2",
```

Do you actually need 7.2? FlexibleMink only requires 7.1.

```suggestion
    "php": "^7.1",
```

---

7.2 is the testing image that's ran

---

what's priority you think?